packages/wallet/src/services/ProviderTracker/TrackedNetworkInfoProviderRequiredByWallet.ts

```diff
@@ -35,23 +27,20 @@ export class NetworkInfoProviderStats {
 /**
  * Wraps a NetworkInfoProvider, tracking # of calls of each function
```

Comment:
```suggestion
 * Wraps a NetworkInfoRequiredByWalletProvider, tracking # of calls of each function
```

Reply:
Fixed the comment in b151972e847831473ed045af83c70d41a8724164